PhotoMapTests/Mocks/Services/LocationServiceMock.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  LocationServiceMock.swift
+//  PhotoMapTests
+//
+//  Created by Dzmitry Makarevich on 19.05.21.
+//
+
+import Foundation
+import CoreLocation
+import Combine
+@testable import PhotoMap
+
+class LocationServiceMock: NSObject, LocationServiceType {
+    var isEnable = CurrentValueSubject<Bool, Never>(false)
+    var status = CurrentValueSubject<CLAuthorizationStatus, Never>(.notDetermined)
+    var location = CurrentValueSubject<CLLocation, Never>(CLLocation.init())
+    
+    var currentCoordinate: CLLocationCoordinate2D {
+        location.value.coordinate
+    }
+
+    lazy var locationManager: CLLocationManager = {
+        let locationManager = CLLocationManager()
+        locationManager.delegate = self
+        locationManager.desiredAccuracy = kCLLocationAccuracyBest
+
+        return locationManager
+    }()
+
+    private let cancelBag = CancelBag()
+
+    override init() {
+        super.init()
+
+        if #available(iOS 14.0, *) {
+            self.status.send(locationManager.authorizationStatus)
+        } else {
+            self.status.send(CLLocationManager.authorizationStatus())
+        }
+
+//        bind()
+        locationManager.startUpdatingLocation()
```

Comment:
No real logic should be placed in mocked/stubbed classes

Reply:
fix 6f58eb2